### ISSUE TITLE
Add YouTube as a recognized link type

### ISSUE THREAD

**Issue by freeatnet** [forkdelta/tokenbase#48]
Eg., https://travis-ci.org/forkdelta/tokenbase/builds/336279526:

```
tokens/0x2a093bcf0c98ef744bb6f69d74f2f85605324290.yaml:
  1) Non-critical issue: can 'YouTube: https://www.youtube.com/watch?v=QMdAEhPkNbA' be better described by one of well-known types: frozenset({'Email', 'Twitter', 'Telegram', 'Website', 'CoinMarketCap', 'Linkedin', 'Facebook', 'Slack', 'Reddit', 'WeChat', 'Blog', 'Bitcointalk', 'Whitepaper', 'Github'})?
```

YouTube cannot be described by the above types, and it's a valid source of information, so we should add it.

Technical: 
* Add `'YouTube'` to `KNOWN_LINK_TYPES` in `scripts/test.py`.
* Done.

**Comment by JonathonDunford** [comment]
We should also add Discord

**Comment by JonathonDunford** [comment]
Fixed in #123
